docs/api/_index.md

```diff
@@ -170,6 +172,28 @@ GET /ha-tracker
 
 Displays a web page with the current status of the HA tracker, including the elected replica for each Prometheus HA cluster.
 
+## Debugger
```

Comment:
Move above, grouped together with the "All services"

Reply:
Done